cms_helper.py

```diff
@@ -23,6 +31,7 @@ def gettext(s):
         'permissions': 10,
     },
     'ROOT_URLCONF': 'tests.test_utils.urls',
+    'MIGRATION_MODULES': DisableMigrations(),
```

Comment:
any reason for disabling migrations during tests?

Reply:
@vipulnarang95 I'm not sure why this has been done and I agree that it probably shouldn't be,

Reply:
@Aiky30
Jonathan proposed this solution since he has made most of the addons 2.2 and CMS4 compatible.

Reply:
You'll get a migration error that originates from django-cms if you don't. It's due to some transaction limitation in sqlite. This only happens in django 2.2 if I remember it correctly.

Reply:
@jonathan-s Thanks. @yakky Is this acceptable if we added a comment as to why this is in place so in the future when  patch or work around is found this can be removed? Maybe we find an issue in the root package to link to in the comment.

Reply:
I'm fine with a `# FIXME` comment to investigate this further

Reply:
@vipulnarang95 Can you please add a comment to this:

`# FIXME: Django CMS migrtions with Django 2.2 produce an error when running tests, temporarily disabling migrations`